_includes/news_banner_info.html

```diff
@@ -0,0 +1,27 @@
+{% assign news_collection_size = site.news.size %}
+
+{% if news_item.order == site.news.size && news_item.order > 1 %}
```

Comment:
You don't need to use `site.news.size` if you're going to use `{% assign news_collection_size = site.news.size %}`. Instead, you can just use `news_collection_size`.